### ISSUE TITLE
fixed nodes - ignored x,y coordinates?

### ISSUE THREAD

**Issue by pocelka** [ogobrecht/d3-force-apex-plugin#15]
I'm still playing with this plugin. Yesterday I spent some time with re-organizing some of the nodes  and pining them to some user friendly position as x and y coordinates are different than in our current C# application (different grid). I just pined few nodes as there is more than 100 node and I just wanted to test fixed position feature. Today I updated the coordinates to database, however it seems like they are ignored.

When query returns true as fixed than nodes are pinned in the grid however with each refresh of the page coordinates are changing. It seems like the coordinates from DB are ignored.

Attributes:
{
  "minNodeRadius": 10,
  "useDomParentWidth": true,
  "lassoMode": false,
  "showLabels": true,
  "zoomMode": true,
  "pinMode": true,
  "colorScheme": "direct",
  "height": 650,
  "debug": true,
  "showTooltips": true,
  "tooltipPosition": "node",
  "labelDistance": 2,
  "zoomMode": true,
  "minZoomFactor": 0.1,
  "transform": {
    "translate": [
      81.23854596788073,
      51.81832738338741
    ],
    "scale": 0.757858283255199
  }
}

Custom Styles:
.net_gobrechts_d3_force_tooltip { width: 500px !important; }
//.net_gobrechts_d3_force text.label{ display: none; }

[data.zip](https://github.com/ogobrecht/d3-force-apex-plugin/files/366962/data.zip)

**Comment by pocelka** [comment]
Shame on me. This issue can be closed. I looked also to some other closed issues and seems like similar one was already reported. I've checked the plugin version and I was using v2.0.0. In 2.0.1 it seems fixed.

I've initially downloaded sample app via https://apex.world/ords/f?p=100:710:8948694552907::::P710_PLG_ID:NET.GOBRECHTS.D3.FORCE

where it seems like there is a link to older demo app version.
